### ISSUE TITLE
Can't install steam via apt on Kali linux 2021.4 (Debian Bullseye)

### ISSUE THREAD

**Issue by revix-0** [ValveSoftware/steam-for-linux#8365]
#### Your system information

* Steam client version (build number or date): none, can't install it
* Distribution (e.g. Ubuntu): Kali (Debian bases)
* Opted into Steam client beta?: [Yes/No] No
* Have you checked for system updates?: [Yes/No] yes

#### Please describe your issue in as much detail as possible:
Treid to install steam via apt after a fresh kali linux install, which contained most of the libraries already, such as llvm. THe package requires me to install libllvm12:i386, and despite the fact adding the architecture i386 with dpkg and doing update, it requires me this:

```gstreamer1.0-gl gstreamer1.0-plugins-bad libegl-dev libgl-dev libgl1
  libgl1-mesa-dev libgl1-mesa-dri libgles-dev libglu1-mesa libglvnd-dev
  libglx-dev libglx-mesa0 libglx0 libgstreamer-gl1.0-0 libllvm12
  libwebkit2gtk-4.0-37 libxatracker2 mesa-utils steam-launcher x11-utils xorg
  xserver-xorg xserver-xorg-core xserver-xorg-input-all
  xserver-xorg-input-libinput xserver-xorg-input-wacom xserver-xorg-video-all
  xserver-xorg-video-amdgpu xserver-xorg-video-ati xserver-xorg-video-fbdev
  xserver-xorg-video-intel xserver-xorg-video-nouveau xserver-xorg-video-qxl
  xserver-xorg-video-radeon xserver-xorg-video-vesa xserver-xorg-video-vmware
  zenity
  ```
basically, it wants to uninstall the entire xorg display, and other essential packages, because of conflicting architecture versions. How can i avoid this?

**Comment by kisak-valve** [comment]
Hello @revix-0, Kali Linux is not supported here because it's contradictory to the fundamental goals of the distribution, which focuses on security testing and is not intended to be used as a general purpose desktop environment.

**Comment by revix-0** [comment]
I would have accepted a more honest request like: we don't know how to fix this yet. But thanks anyways.
